packages/code-du-travail-frontend/src/layout/Footer.tsx

```diff
@@ -375,7 +373,7 @@ const StyledLink = styled.a.attrs((props) => ({
   font-weight: normal;
   text-decoration: none;
   cursor: pointer;
-  padding: ${spacings.tiny} 0;
+  padding: ${spacings.xsmall} 0;
```

Comment:
j'ai ajouté un peu de padding pour fixer cette erreur : [Tap targets are not sized appropriately](https://socialgouv.github.io/dashlord-fabrique/report/aHR0cHM6Ly9jb2RlLnRyYXZhaWwuZ291di5mcg==/lhr.html#seo)

Reply:
ok cool, ça ! 🔥